Apromore-Clients/manager-security/src/main/java/org/apromore/security/filter/SecurityPrincipal.java

```diff
@@ -0,0 +1,63 @@
+/*-
+ * #%L
+ * This file is part of "Apromore Core".
+ * %%
+ * Copyright (C) 2018 - 2021 Apromore Pty Ltd.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+package org.apromore.security.filter;
+
+import java.security.Principal;
+
+public class SecurityPrincipal implements Principal, java.io.Serializable {
+
+	private String username;
+
+    public SecurityPrincipal(String username) {
+        if (username == null) throw new NullPointerException("illegal null input");
+
+        this.username = username;
+    }
+
+    public String getName() {
```

Comment:
A little weird that the field is "username" and the property is "name".

Reply:
Refactored to consistent field/method/property name "name" [DONE]